app/styles/layouts/settings.css

```
@@ -597,59 +597,109 @@
+    overflow-y: scroll;
+    margin: -40px -40px 0;
+    padding: 40px 40px 0;
 }
 
-.theme-validation-errors p {
-    margin-bottom: 0;
+.theme-validation-item {
+    margin: 12px 0 0;
+    padding: 12px 16px;
+    border: 1px solid #e5eff5;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    background: var(--whitegrey);
+    border: 1px solid var(--lightgrey);
 }
 
-.theme-validation-errordescription {
-    margin-top: 1em;
-    margin-bottom: 0.5em;
+.theme-validation-item h4 {
+    margin: 0;
+    font-size: 1.4rem;
+    font-weight: 400;
+    line-height: 1.5em;
 }
 
-.theme-validation-errordescription span {
-    font-weight: 600;
+.theme-validation-rule-text {
+    flex-grow: 1;
 }
 
-.theme-validation-errortype {
-    font-size: 1.8rem;
+.theme-validation-item.theme-fatal-error {
+    background: color-mod(var(--red) alpha(0.04));
+    border: 1px solid color-mod(var(--red) alpha(0.4));
+}
+
+.theme-validation-item.theme-fatal-error .theme-validation-rule-text::before,
+.theme-validation-item.theme-error .theme-validation-rule-text::before,
+.theme-validation-item.theme-warning .theme-validation-rule-text::before
+{
+    font-weight: 600;
 }
 
-.theme-validation-errortype.fatal {
+.theme-validation-item.theme-fatal-error .theme-validation-rule-text::before {
+    content: "Fatal error:";
     color: var(--red);
 }
 
-.theme-validation-item {
-    margin: 0 0 15px;
-    padding: 15px 20px;
-    border: 1px solid #e5eff5;
-    border-radius: 5px;
-    display: flex;
-    flex-direction: column;
+.theme-validation-item.theme-error .theme-validation-rule-text::before {
+    content: "Error:";
 }
 
-.theme-validation-item h4 {
-    margin: 0;
-    font-size: 1.5rem;
-    font-weight: 500;
+.theme-validation-item.theme-warning .theme-validation-rule-text::before {
+    content: "Warning:";
+}
+
+.theme-fatal-error .theme-validation-type-label::before,
+.theme-error .theme-validation-type-label::before,
+.theme-warning .theme-validation-type-label::before {
+    content: "";
+    display: block;
+    border-top-right-radius: 2px;
+    border-bottom-right-radius: 2px;
+    width: 8px;
+    height: 16px;
+    margin-top: 3px;
+    margin-left: -17px;
+}
+
+.theme-fatal-error .theme-validation-type-label::before, 
+.theme-error .theme-validation-type-label::before {
+    background: color-mod(var(--red) alpha(0.85));
+}
+
+.theme-warning .theme-validation-type-label::before {
+    background: color-mod(var(--yellow));
+}
+
+.theme-validation-list ul {
+    list-style: disc;
+}
+
+.theme-validation-list code,
+.theme-validation-rule-text code {
+    font-size: 0.9em;
 }
 
 .theme-validation-item h6 {
-    font-size: 1.4rem;
+    font-size: 1.3rem;
     font-weight: 500;
 }
 
+.theme-validation-item {
+    background: var(--whitegrey-l2);
+}
+
 .theme-validation-toggle-details {
     display: flex;
     justify-content: space-between;
     flex-grow: 1;
-    align-items: center;
+    align-items: start;
```

`autoprefixer` is complaining about this because it has "mixed support" and recommends `flex-start` instead. Looking at [the flexbox spec](https://www.w3.org/TR/css-flexbox-1/#align-items-property), `start` isn't listed as one of the possible values.

```suggestion
    align-items: flex-start;
```